src/Searcher/SearcherInterface.php

```diff
@@ -140,4 +140,11 @@ public function truncateWatches();
      * @return array array of stats
      */
     public function stats();
+
+    /**
+     * Get all the known server names.
+     *
+     * @return array|false array of server names or false if not supported
```

Comment:
i think `array|null` is better typing, then you can use `?array` as native return type

Reply:
I didn't add native type because it's not used in other methods

Reply:
please add the type as asked.

current minimum PHP version is 7.2 so all language features compatible with that version can be used.

Reply:
Updated